tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -647,10 +655,15 @@ def context():
 
 
 @pytest.fixture
-def section_groups_on(context):
+def canvas_sections_on(context):
```

Comment:
Remaded this one, having `groups` in the name was a bit confusing with the new feature